incubator/jaeger/templates/agent-ds.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: "{{ template "jaeger.fullname" . }}-agent"
+  labels:
+    app: "{{ template "jaeger" . }}"
+    jaeger-infra: agent-daemonset
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    component: "{{ template "jaeger.fullname" . }}-agent"
```

Comment:
Don't add the fullname to the component name. Should just be `agent`.

Reply:
I mirrored the component names based on what is recommended from the Jaeger repo for Kubernetes. Their best practice is to have jaeger-agent as seen here:
https://github.com/jaegertracing/jaeger-kubernetes/blob/master/production/jaeger-production-template.yml

I can make the component name static meaning replace "{{ template "jaeger.fullname" . }}-agent" to "jaeger-agent".

Reply:
Changed to static names.